### PR TITLE
Update Javadoc of premium SDK for azure-resourcemanager-containerservice

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/ContainerServiceManager.java [hunk 1]
```diff
@@ -87,7 +87,11 @@ private ContainerServiceManager(HttpPipeline httpPipeline, AzureProfile profile)
                 .buildClient());
     }
 
-    /** @return the Azure Kubernetes cluster resource management API entry point */
+    /**
+     * Gets he Azure Kubernetes cluster resource management API entry point.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/ContainerServiceManager.java:91]: the

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/AgentPool.java [hunk 2]
```diff
@@ -13,86 +13,178 @@
...
+     * Gets the tags of the agents.
+     *
      * @return the tags of the agents.
      */
     Map<String, String> tags();
 
     /**
+     * Checks whether FIPS-enabled OS is been used for agent pool's machines.
+     *
      * @return whether FIPS-enabled OS is been used for agent pool's machines
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/AgentPool.java:188]:
```suggestion
     * Checks whether FIPS-enabled OS is being used for agent pool's machines.
     *
     * @return whether FIPS-enabled OS is being used for agent pool's machines
```

File: sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/ContainerServiceResourceTypes.java [hunk 3]
```diff
@@ -23,7 +32,11 @@ public static ContainerServiceResourceTypes fromString(String name) {
         return fromString(name, ContainerServiceResourceTypes.class);
     }
 
-    /** @return known ExtendedLocationTypes values. */
+    /**
+     * Gets known ExtendedLocationTypes values..
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/ContainerServiceResourceTypes.java:36]:
```suggestion
     * Gets known ExtendedLocationTypes values.
```

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/ContainerServiceResourceTypes.java]: Fixed by the [commit#ba36bfb](https://github.com/Azure/azure-sdk-for-java/pull/43518/commits/ba36bfbec20ab3096bf1edbdbdc4e5f2d1ade40d)

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**